src/components/counter/Counter.tsx

```diff
@@ -0,0 +1,30 @@
+import { useState } from 'react';
+import styles from './counter.module.scss';
+import PlusIcon from '../icons/PlusIcon';
+import MinusIcon from '../icons/MinusIcon';
+
+function Counter() {
+  const [count, setCount] = useState(0);
+
+  const increment = () => {
+    setCount((prev) => prev + 1);
+  };
+
+  const decrement = () => {
+    setCount((prev) => (prev > 0 ? prev - 1 : 0));
+  };
+
+  return (
+    <div className={styles['counter']}>
+      <button className={styles['decrement']} onClick={decrement}>
+        <MinusIcon />
+      </button>
+      <div className={styles['count']}>{count}</div>
+      <button className={styles['increment']} onClick={increment}>
```

Comment:
I recommend creating a separate component called `IconButton`, where you can add an icon as a prop.

```js
<IconButton icon={<MinusIcon />} />
```

This way you can style it once and then reuse multiple times in the project.

As an inspiration you can take a look at chakra-ui similar component https://v2.chakra-ui.com/docs/components/icon-button/usage